apps/frontend/src/modules/projects/slices/project.slice.ts

```diff
@@ -4,30 +4,42 @@ import { DataStatus } from "~/libs/enums/enums.js";
 import { type ValueOf } from "~/libs/types/types.js";
 import { type ProjectGetAllItemResponseDto } from "~/modules/projects/projects.js";
 
-import { loadAll } from "./actions.js";
+import { create, loadAll } from "./actions.js";
 
 type State = {
-	dataStatus: ValueOf<typeof DataStatus>;
+	projectCreateStatus: ValueOf<typeof DataStatus>;
 	projects: ProjectGetAllItemResponseDto[];
+	projectsStatus: ValueOf<typeof DataStatus>;
 };
 
 const initialState: State = {
-	dataStatus: DataStatus.IDLE,
+	projectCreateStatus: DataStatus.IDLE,
 	projects: [],
+	projectsStatus: DataStatus.IDLE,
 };
 
 const { actions, name, reducer } = createSlice({
 	extraReducers(builder) {
 		builder.addCase(loadAll.pending, (state) => {
-			state.dataStatus = DataStatus.PENDING;
+			state.projectsStatus = DataStatus.PENDING;
```

Comment:
for get all let's leave it as dataStatus for now, to keep it the same as in other places
```suggestion
			state.dataStatus = DataStatus.PENDING;
```